test/flop_phoenix_test.exs

```diff
@@ -1928,6 +1948,42 @@ defmodule Flop.PhoenixTest do
       assert Jason.decode!(phx_click) == [["push", %{"event" => "sort"}]]
     end
 
+    test "application of custom sort directions per column" do
+      html =
```

Comment:
The test table functions are there for historical reasons, but I don't like them very much, since it's hard to see what's going on when you're reading the test. Would you mind rewriting the test similar to this one? https://github.com/woylie/flop_phoenix/pull/245/files#diff-6448af2f66e7af4d5d00b84580bd8eec3f6e3dfa9b66efee6087c8ecbed92ebfR1822

Reply:
addressed in fe30929029b0af14f79c92a7caecba50b799dea0